qdrant-landing/content/articles/vector-search-filtering.md

```diff
@@ -0,0 +1,726 @@
+
+We know that there are three possible laptops that suit our price point. 
+Let's see how Qdrant's filterable vector index works and why it is the best method of capturing all available results.  
+
+First, add five new laptops to your online store. Here is a sample input:
+
+```python
+laptops = [
+    (1, [0.1, 0.2, 0.3, 0.4], {"price": 899.99, "category": "laptop"}),
+    (2, [0.2, 0.3, 0.4, 0.5], {"price": 1299.99, "category": "laptop"}),
+    (3, [0.3, 0.4, 0.5, 0.6], {"price": 799.99, "category": "laptop"}),
+    (4, [0.4, 0.5, 0.6, 0.7], {"price": 1099.99, "category": "laptop"}),
+    (5, [0.5, 0.6, 0.7, 0.8], {"price": 949.99, "category": "laptop"})
+]
+```
+
+The four-dimensional vector can represent features like laptop CPU, RAM or battery life, but that isn’t specified. The payload, however, specifies the exact price and product category.
+
+Now, set the filter to "price is less than $1000":
+
+```json
+{
+  "key": "price",
+  "range": {
+    "gt": null,
+    "gte": null,
+    "lt": 1000,
+    "lte": null
+  }
+}
+```
+
+When a price filter of < $1000 is applied, vector search returns the following results:
+
+```json
+[
+  {
+    "id": 3,
+    "score": 0.9978443564622781,
+    "payload": {
+      "price": 799.99,
+      "category": "laptop"
+    }
+  },
+  {
+    "id": 1,
+    "score": 0.9938079894227599,
+    "payload": {
+      "price": 899.99,
+      "category": "laptop"
+    }
+  },
+  {
+    "id": 5,
+    "score": 0.9903751498208603,
+    "payload": {
+      "price": 949.99,
+      "category": "laptop"
+    }
+  }
+]
+```
+
+As you can see, Qdrant's filtering method has a greater chance of capturing all possible search results. 
+
+This specific example uses the `range` condition for filtering. Qdrant, however, offers many other possible ways to structure a filter
+
+**For detailed usage examples, [filtering](/documentation/concepts/filtering/) docs are the best resource.** 
+
+### Scrolling instead of searching
+
+You don't need to use our `search` and `query` APIs to filter through data. The `scroll` API is another option that lets you retrieve lists of points which meet the filters.
+
+In Qdrant, scrolling is used to iteratively **retrieve large sets of points from a collection**. It is particularly useful when you’re dealing with a large number of points and don’t want to load them all at once. Instead, Qdrant provides a way to scroll through the points **one page at a time**.
+
+You start by sending a scroll request to Qdrant with specific conditions like filtering by payload, vector search, or other criteria.
+
+Let's retrieve a list of top 10 laptops in the store: 
+
+```http
+POST /collections/online_store/points/scroll
+{
+    "filter": {
+        "must": [
+            {
+                "key": "category",
+                "match": {
+                    "value": "laptop"
+                }
+            }
+        ]
+    },
+    "limit": 10,
+    "with_payload": true,
+    "with_vector": false
+}
+```
```

Comment:
I don't think this example makes sense, because this will not give us the top 10 laptops.

The reason is that we don't define how to order results here, so we just get any 10 laptops.

We could change this example to sort by price using [`order_by`](https://qdrant.tech/documentation/concepts/points/#order-points-by-payload-key). That would give us the '10 cheapest laptops`.

Alternatively we might change this example into getting all laptops. But I like the above better.